app/experimenter/nimbus-ui/src/components/AppErrorBoundary/index.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { Alert } from "react-bootstrap";
+import sentryMetrics from "../../services/sentry";
+
+class AppErrorBoundary extends React.Component {
+  state: {
+    error: undefined | Error;
+  };
+
+  constructor(props: {}) {
+    super(props);
+    this.state = { error: undefined };
+  }
+
+  static getDerivedStateFromError(error: Error) {
+    return { error };
+  }
+
+  componentDidCatch(error: Error) {
+    sentryMetrics.captureException(error);
+  }
+
+  render() {
+    const { error } = this.state;
+    return error ? <AppErrorAlert {...{ error }} /> : this.props.children;
+  }
+}
+
+export const AppErrorAlert = ({ error }: { error: Error }) => {
```

Comment:
On further looking at `AppErrorDialog` I thought of a few things:

1. Only AppErrorBoundary ever uses it, so I moved it to live with that component
1. We have react-bootstrap now, so I just used `<Alert>` for styling
1. Used "Alert" rather than "Dialog" because, well, it's an alert I guess?

Reply:
Looks like Storybook build worked:

https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/ca2467d4dc1e023c4283640e8622c23c132a743b/nimbus-ui/index.html?path=/story/components-apperroralert--basic